### PR TITLE
Made the endpoint filter relation 1-1

### PR DESCRIPTION
**Pull request by ShimShtein**

### CODE REVIEW

File: test/factories/filter_factory.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :filter, :class => ::Filter do
+  end
```
Comment (adamruzicka) [inline test/factories/filter_factory.rb]: Is this block missing a body?

Comment (ShimShtein) [reply test/factories/filter_factory.rb]: I want the factory to exist, but I don't set up any properties. I left the block to be more consistent with other definitions.


I can remove the empty block if it looks annoying for you.

Comment (adamruzicka) [reply test/factories/filter_factory.rb]: It just felt forgotten. I'm ok with it if it is intentional

**Comment by adamruzicka** [comment]
Seeds are currently failing with
```
ActiveModel::UnknownAttributeError: unknown attribute 'endpoints' for Filter.
notifications-backend/db/seeds.rb:27:in `block in <main>'
notifications-backend/db/seeds.rb:7:in `<main>'
```

**Comment by ShimShtein** [comment]
@adamruzicka Finally it's ready for review.

**Comment by adamruzicka** [comment]
Thanks @ShimShtein !

**Pull request merged**